.npmrc

```diff
@@ -0,0 +1,2 @@
+@fortawesome:registry=https://npm.fontawesome.com/
+//npm.fontawesome.com/:_authToken=${FONTAWESOME_NPM_AUTH_TOKEN}
```

Comment:
@atlwendy you will need to add the `FONTAWESOME_NPM_AUTH_TOKEN` to your local tokens config (just like we do for nx cloud)

Reply:
Did that already :)